protoc-gen-swagger/genswagger/template_test.go

```diff
@@ -693,8 +693,8 @@ func TestTemplateToSwaggerPath(t *testing.T) {
 	}{
 		{"/test", "/test"},
 		{"/{test}", "/{test}"},
-		{"/{test=prefix/*}", "/{test}"},
-		{"/{test=prefix/that/has/multiple/parts/to/it/*}", "/{test}"},
+		{"/{test=prefix/*}", "/prefix/*"},
```

Comment:
This isn't correct anymore, is it? I don't think we should leave any `*`in the path.